### ISSUE TITLE
Errors when there are no paths

### ISSUE THREAD

**Issue by simoncozens** [RafalBuchner/StemThickness#4]
Here's the logic leading up to line 460:

```python
            closestPathTime = None
            dist = 100000
            for path in layer.paths:
                # stuff
            n = math.floor(closestPathTime)
```

To avoid spamming the error log, you want something like:

```python
            if not closestPathTime:
                return
```

in there.

**Comment by RafalBuchner** [comment]
Thanks Simon, I will look into that

**Comment by simoncozens** [comment]
Merged with #6.
